vaadin-upload-flow-parent/vaadin-upload-flow/src/main/java/com/vaadin/flow/component/upload/Upload.java

```diff
@@ -649,6 +650,13 @@ private String getStringObject(String propertyName, String object,
         return result;
     }
 
+    /**
+     * Clear the filelist of the component.
```

Comment:
```suggestion
     * Clear the fileList of the component.
```

Reply:
Why do we want to keep this as one word in the documentation? I don't think "fileList" is a known concept in `Upload`.